src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/diagnostics/IJavaDiagnosticsParticipant.java

```diff
@@ -24,10 +24,8 @@
  *
  */
 public interface IJavaDiagnosticsParticipant {
-
-	// Not used in liberty-tools-intellij:
-	// public static final ExtensionPointName<IJavaDiagnosticsParticipant> EP_NAME = ExtensionPointName.create("open-liberty.intellij.javaDiagnosticsParticipant");
-
+	ExtensionPointName<IJavaDiagnosticsParticipant> EP_NAME =
+			ExtensionPointName.create("open-liberty.intellij.javaDiagnosticsParticipant");
```

Comment:
@aparnamichael @turkeylurkey same question here

Reply:
@TrevCraw @turkeylurkey @aparnamichael The extension point in LTI is [JavaDiagnosticsDefinition](https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/internal/core/java/corrections/JavaDiagnosticsDefinition.java) which was introduced for the same reasons as `JavaCompletionDefinition`.